frontend/packages/console-plugin-sdk/README.md

```diff
@@ -0,0 +1,234 @@
+# OpenShift Console Plugins
+
+Openshift Console Plugins extend the functionality of the console. Plugins are enabled by
+adding them as `dependencies` of the [console-app](frontend/packages/console-app) package.
+
+- Plugins are static, their code gets bundled ("baked") into Console build output:
+  - Plugins are not loaded dynamically at runtime, which in turn simplifies the architecture (no need to sandbox plugin execution or its interaction with Console app or Bridge server).
+  - Console-app dependencies control the set of plugins to include in Console build, this can be overridden via CONSOLE_PLUGINS env. variable.
+- Plugins have their code represented as Console monorepo packages:
+  - This increases the level of trust between Console app and its plugins.
+  - Plugin code close to Console code avoids cross-repo PR maintenance issues.
+  - Each plugin has an OWNERS file denoting its ownership (see here for details on reviewers and approvers).
+- Plugins are declarative, meaning they provide a list of extensions to be interpreted:
+  - Each extension has a unique type and properties representing its parameters (data and/or callbacks).
+  - Each extension has flags object to support gating by Console feature flags.
+- Console demo plugin is used to demonstrate specific extensions:
+  - It should contain at least one instance of every extension type (sadly, this isn't the case yet).
+  - It should be a "synthetic" (foo-bar, not-a-real-world) example acting as a starting point for further exploration.
+
+See the [demo plugin](frontend/packages/console-demo-plugin) for reference.
+
+## Directory and File Names
+
+Plugin packages should be in [frontend/packages](frontend/packages) directory.
+
+- Every plugin should have `package.json` and `OWNERS` files.
+- Every plugin should follow the recommended file structure:
+  - `src` directory.
+  - `integration-tests` directory (if needed).
+  - Plugin entry module at `src/plugin.ts`.
+  - `src/components` for React components, `src/models` for k8s model definitions.
+  - Unit tests co-located at `</path/to/unit>/__tests__/unit.spec.ts`.
+
+Unit tests are encouraged, but optional. Unit tests should be placed in __tests__ directories as close as possible to the source code that they test. Integration tests should be placed in an integration-tests directory at the root level of the plugin directory.
```

Comment:
`__tests__` must be formatted as `monospaced` since the underscore has special meaning in Markdown syntax.

> I think we should keep the comment about unit tests being encouraged, but the remaining information about unit and integration test locations are already covered in the list and can be removed.

If it makes sense, we can simply refer to the section that documents locations. Otherwise, yes, we should avoid duplicity as it makes the doc maintenance harder.